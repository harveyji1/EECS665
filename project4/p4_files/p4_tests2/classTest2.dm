MyClass : class {
  x : int;
  y : bool;
  myFunction : (param : int) bool {
    give y;
  }
};