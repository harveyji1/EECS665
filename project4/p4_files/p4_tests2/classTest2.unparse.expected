MyClass{MyClass} : class {
    x{int} : int;
    y{bool} : bool;
    myFunction{(int)->bool} : (param{int} : int) bool  {
        give y{bool};
    }
};
